### DISCUSSION TITLE
hfs v0.41 import accounts from xlsx or csv

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#170] [Ideas]
can you enable option import accounts from csv or xlsx file? and one more hide admin panel from public access /~/admin?

**Comment by rejetto** [comment]
would it be ok if first column is username, then password, and optional third column is group?

**Reply by gogi100** [reply]
yes, it would be ok.

**Comment by rejetto** [comment]
next version
<img width="666" alt="image" src="https://user-images.githubusercontent.com/1367199/227055980-a8a0bd25-407d-4eb0-a52c-34fad2a45bd3.png">

**Reply by gogi100** [reply]
this option exists in the current version of hfs 0.41 or wil public in next version?

**Reply by rejetto** [reply]
above the image you can read "next version" 😊

**Comment by rejetto** [comment]
please, try the new csv feature
https://github.com/rejetto/hfs/discussions/179

**Comment by gogi100** [comment]
one more requirements: i need that the csv file for importing contains next
columns: username, password, redirect (location where user will be redirect)

**Comment by rejetto** [comment]
the beta3 i gave you has this feature too.
Is it working?

**Comment by gogi100** [comment]
Thanks
